app/models/orchestration_stack.rb

```diff
@@ -85,6 +85,22 @@ def raw_update_stack(_template, _options = {})
     raise NotImplementedError, _("raw_update_stack must be implemented in a subclass")
   end
 
+  def update_stack_queue(userid, template, options = {})
```

Comment:
@bzwei I've updated the ems_infra_controller scaling/scaledown actions to call this rather than raw_update_stack (as part of a larger mandate to remove any direct provider calls from UI controllers) -- you can see that in https://github.com/ManageIQ/manageiq-ui-classic/pull/373

As for create/delete, it would make sense to add those, but so far (across all models) we have only added _queue methods for those provider actions that are actually called from the UI. At the moment nothing in the UI calls create or delete on orchestration stacks.